code-challenges.js

```diff
@@ -10,36 +10,84 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a conditional statement that evaluates two strings and determines which of the strings has more characters. Use the two sets of test variables provided.
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+// // // Expected outcome: "banana"
+// if(fruit1.length > fruit2.length){
+//     console.log(fruit1)
+// }else if(fruit2.length > fruit1.length){
+//     console.log(fruit2)
+// }
+// conditional statement to compare which string is longer based on index
+// first console.log outputs fruit1 if it is longer
+// second console.log outputs fruit2 if it is longer
+// compares the index of the 2 strings
 
 // Set two:
 // const fruit1 = "cherry"
 // const fruit2 = "kiwi"
-// Expected outcome: "cherry"
+
+// const fruit1 = "cherry"
+// const fruit2 = "kiwi"
+// // Expected outcome: "cherry"
+// if(fruit1.length > fruit2.length){
+//     console.log(fruit1)
+// }else if(fruit2.length > fruit1.length){
+//     console.log(fruit2)
+// }
 
 // Pseudo code:
+// determine what data type we are working with
+// set a condition which compares data types which are 2 strings
+// (.length) is used to count the number of index there is in our string
+// we are using "if" statements as our conditional statements to get a bolean equivalent from the 2 items being compared
+// (console.log) is the method we are using to output our statement which is attatched to a data type that is a variable so we can replace true or false with said data type to get the content that is assigned to the variable given to us
+// else if is for the other variable that we have if the 2nd variables index is larger
+
 
 
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+ const padresWorldSeriesRuns =(padres1984WorldSeriesRuns.length) + (padres1998WorldSeriesRuns.legnth)
+ console.log(padresWorldSeriesRuns)
+
```

Comment:
Though this gives you the expected output, the code is not doing what the prompt is asking. We need to combined the arrays first before we get the length. When refactoring think about what build-in method that we learned about that will help you accomplish this. Remember pay close attention to what the prompt is asking of you.